src/components/AddAccommodationForm.js

```diff
@@ -0,0 +1,302 @@
+import React, { Component } from "react";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+import styles from "../styles/signup.module.css";
+import Grid from "@material-ui/core/Grid";
+import MenuItem from "@material-ui/core/MenuItem";
+import countryList from "country-list";
+var cities = require("countries-cities");
+import Paper from "@material-ui/core/Paper";
+import PublishIcon from "@material-ui/icons/Publish";
+import Snackbar from "@material-ui/core/Snackbar";
+import IconButton from "@material-ui/core/IconButton";
+import CloseIcon from "@material-ui/icons/Close";
+import PropTypes from "prop-types";
+import { withRouter } from "react-router-dom";
+import CircularProgress from "@material-ui/core/CircularProgress";
+
+class addAccommodationForm extends Component {
+  state = {
+    name: "",
+    city: "",
+    country: "RW",
+    city: "",
+    imageUrl: "",
+    around: [],
+    amenities: [],
+    errors: {},
+    open: false,
+    loading: "none",
+    btn: "Add",
+    isImageUploaded: true,
+    profileWidget: cloudinary.createUploadWidget(
+      {
+        cloudName: "octopusbn",
+        uploadPreset: "barefootNomadCloud",
+        multiple: false,
+        cropping: true,
+        croppingShowBackButton: true,
+      },
+      (error, result) => {
+        if (!error && result && result.event === "success") {
+          const data = { ...this.state };
+          this.state.imageUrl = result.info.secure_url;
+          this.setState({ data });
+          this.setState({ ...this.state, isImageUploaded: true });
+        }
+      }
+    ),
+  };
+
+  onChange = (e) =>
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+
+  onSubmit = (e) => {
+    e.preventDefault();
+    const amenity = this.state.amenities.toString().split(",");
+    const arounds = this.state.around.toString().split(",");
```

Comment:
doesn't this mean that if  a user doesn't put in , the whole string will just be counted as one
if possible please try to split using the newline 

example `str.split("\n")`

Reply:
@blaise82 yeah that is the intention, if a user doesn't put a comma that will be counted as one amenity or around(for example: Inema art center, this is one around of your accommodation), cos again if I split using new line and a user enter a block of words but on the same line they will still be counted as one